client/src/components/CTACard.tsx

```diff
@@ -0,0 +1,263 @@
+import * as React from 'react';
+
+import Grid from '@mui/material/Grid';
+import Avatar from '@mui/material/Avatar';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import { IconButton, Typography } from '@mui/material';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { makeStyles } from '@mui/styles';
+
+//     backgroundColor: '#37718E', //primary dark blue color
+const useStyles = makeStyles((theme) => ({
+  CTAGrid: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#FFFFFF',
+    paddingTop: '25px',
+  },
+
+  CTACardLayout: {
+    border: '1',
+    borderColor: '#37718E',
+  },
+
+  CTACardContent1: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#37718E',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  CTACardContent2: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#7C4164',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  CTACardContent3: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#C33D54',
+    color: '#FFFFFF',
+    fontSize: '18px',
+    fontWeight: 'bold',
+    textAlign: 'left',
+  },
+
+  HeaderStyle: {
+    paddingY: '2',
+    my: '2',
+    mt: '2',
+  },
+
+  ParagraphStyle: {
+    textColor: '#FFFFFF',
+    fontSize: '10',
+    textAlign: 'left',
+    width: '80%',
+  },
+
+  CTAButton: {
+    display: 'inline-block',
+    justifyContent: 'bottom',
+    marginTop: '100px',
+    variant: '',
+    width: '60%',
+  },
+
+  BottomCTACard: {
+    display: 'flex',
+  },
+}));
+
+function CTACard() {
```

Comment:
#### Recommended change
I recommend that you include the entire name `CallToActionCard()`

Reply:
Complete